### PR TITLE
Add basic tests

### PR DESCRIPTION
**Pull request by nterray**
For better documentation of what is going during the conversion and to
ensure that we don't break things with future contributions.

Use Travis CI to run tests.

### CODE REVIEW

File: tests/_fixtures/SetUpTearDownTest.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        // initialize stuff
+        $this->stuff = 'foo';
+    }
+
+    public function tearDown()
+    {
+        // clean
+        $this->stuff = '';
+    }
+}
```
Comment (gorkat) [inline tests/_fixtures/SetUpTearDownTest.php]: Linter says newline is missing

File: tests/_expected/SetUpTearDownTest.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        $this->stuff = 'foo';
+    }
+
+    public function tearDown()
+    {
+        // clean
+        $this->stuff = '';
+        parent::tearDown();
+    }
+}
```
Comment (gorkat) [inline tests/_expected/SetUpTearDownTest.php]: Linter says newline is missing

File: Readme.md [hunk 3]
```diff
@@ -3,6 +3,8 @@ SimpleTest to Mockery
 
 An assistant to ease migration from SimpleTest mocks to Mockery ones.
 
+[![Build Status](https://travis-ci.org/Enalean/simpletest2mockery.svg?branch=master)](https://travis-ci.org/Enalean/simpletest2mockery)
```
Comment (gorkat) [inline Readme.md]: Nice :+1:

File: tests/_fixtures/SetUpTearDownTest.php (hunk 1 above)
Comment (nterray) [reply tests/_fixtures/SetUpTearDownTest.php]: Which linter? Here I have a bunch of fixtures files that mimic legacy code which does not necessarily respect our latest standards. There is no need to pass those files to the linter.

**Pull request merged**